.env.example

```
@@ -37,6 +37,50 @@ LITELLM_MASTER_KEY=
 # Generate a secure key: openssl rand -hex 32
 WEBUI_SECRET_KEY=
 
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)